src/utils/shadowBox.js

```diff
@@ -0,0 +1,197 @@
+import * as twgl from 'twgl.js';
+import { gl } from '../constants';
+import { deg2rad } from './math';
+import { Matrix4, Quaternion, Vector3 } from 'three';
+
+// NOTE (Joseph): this only exists so that i dont have to type 75 then remember what it means
+function fov() {
+  return 75;
+}
+
+// TODO change the near plane to 0.1
+// NOTE (Joseph): this only exists so that i dont have to type 1 then remember what it means
+function nearPlane() {
+  return 1;
+}
+
+function aspectRatio() {
+  return gl.canvas.width / gl.canvas.height;
+}
+
```

Comment:
Might just be better to use constants over functions here?

Reply:
`aspectRatio` cannot be a constant as it *may* change. The others are functions because we don't have the corresponding value as a constant somewhere else. It would be a good thing to later clean those up (both in index and here)

Reply:
Makes sense, we could probably benefit from adding the camera to the manager so we can do something like `manager.camera.fov`. I remember you mentioned something like that earlier but I didn't think we needed it at the time.

Reply:
Yea. We can definitely add that as a future improvement and cleanup.

Reply:
Resolving, fov and near now pulled from camera